packages/formation-react/src/components/Telephone/Telephone.jsx

```diff
@@ -114,15 +143,14 @@ function Telephone({
   const parsedNumber = parseNumber(contact.toString());
   const phoneNumber = CONTACTS[parsedNumber] || parsedNumber;
 
-  // Capture "911" pattern here
-  const contactPattern = pattern || PATTERNS[parsedNumber] || PATTERNS.DEFAULT;
-  const patternLength = contactPattern.match(/#/g).length;
-  const formattedNumber = formatTelText(phoneNumber, contactPattern, extension);
+  // Capture 3 digit patterns here
+  const contactPattern = deriveContactPattern(pattern, parsedNumber);
+  const formattedNumber = formatTelText(phoneNumber, contactPattern);
 
-  if (!phoneNumber || phoneNumber.length !== patternLength) {
-    throw new Error(
-      `Contact number "${phoneNumber}" does not match the pattern (${contactPattern})`,
-    );
+  // Show nothing if no phone number was provided.
+  if (!phoneNumber) {
+    console.warn('Contact number is missing so the <Telephone /> component did not render.');
+    return null;
```

Comment:
Would this be a good use case for React prop-types?

Reply:
It would! Thanks! I'll add that! 🙂

Reply:
Oh wait sorry there are already prop types for the props that get passed in, `phoneNumber` isn't an exact prop (it's just a variable that's constructed during render) and `contact` is already a required prop, so we should be good as far as prop types I think 🙂

Reply:
What if we had it render as  _**Secret phone number**_

Reply:
hahaha **mysterious digits**